tilp/trunk/help/chapter07.html

```diff
@@ -138,15 +138,15 @@ <h1>7. Command line</h1>
 
 
 CONFIG FILE FORMAT
-       This section describes the format of the .tilp config file which is  in
-       the  home  directory  of the user (Linux) or the tilp.ini file (Win32).
-       The file is separated  into  several  sections  (hardware,  gui,  calc,
-       external  programs,  fonts,  screen, misc). But, there is no order, you
-       can write lines as you want.  A line preceded by the '#'  symbol  is  a
-       comment.  Each  line has the following format: option=something.  Every
-       line can be put in any order. Beware: any comment  added  to  the  file
-       will  be overwritten !  Below is described each option and the possible
-       values:
+       This section describes the format of the tilp.ini config file which is
+       in  the  configuration  directory of  the  user  ($XDG_CONFIG_HOME  or
+       LocalAppData on windows). The  file is separated into several sections 

```

Comment:
After merging the PR, a bit later, I'll fix "windows" to "Windows", and add a note about the automatic upgrade from the config file at the old location. Unless you beat me to rewriting the commit :)

Reply:
I fixed this in [0b06ebc](https://github.com/debrouxl/tilp_and_gfm/pull/75/commits/0b06ebccb952f26a2665a5ea57704b01503c7918) :smiley"